src/main/groovy/com/jfrog/bintray/gradle/tasks/BintrayUploadTask.groovy

```diff
@@ -539,34 +540,21 @@ class BintrayUploadTask extends DefaultTask {
     }
 
     Artifact[] collectArtifacts(Publication publication) {
-        if (!publication instanceof MavenPublication) {
+        if (!publication instanceof MavenPublicationInternal) {
             logger.info "{} can only use maven publications - skipping {}.", path, publication.name
             return []
         }
-        def artifacts = publication.artifacts.findResults {
-            boolean signedArtifact = (it instanceof org.gradle.plugins.signing.Signature)
-            def signedExtension = signedArtifact ? it.toSignArtifact.getExtension() : null
```

Comment:
Question - With the new API, we no longer need this?
This code checks whether this is artifact is the signature file, and adds it to the list of uploaded artifact,s with the signature extension.

Reply:
toSignArtifact was deprecated and removed, this PR would also fix #300

Reply:
@eyalbe4 so, to answer your question: yes, we no longer need this.
In fact the current code is wrong as currently publishing signed artifacts does not work at all (#300 ).
Would be great to have this merged and released.

Reply:
I confirm as well that these changes still properly publish the signature files.